scripts/search/update-youtube-index.js

```diff
@@ -0,0 +1,52 @@
+require('dotenv').config();
+const fetch = require('node-fetch');
+const algoliasearch = require('algoliasearch');
+
+const client = algoliasearch(
+    process.env.REACT_APP_ALGOLIA_APP_ID,
+    process.env.REACT_APP_ALGOLIA_ADMIN_KEY,
+);
```

Comment:
DRY, we have this client initialization several times, we should put in a file and reuse